.sage/main.go

```diff
@@ -51,6 +52,16 @@ func GoLint(ctx context.Context) error {
 	return sggolangcilint.Run(ctx)
 }
 
+func GoLintFix(ctx context.Context) error {
+	sg.Logger(ctx).Println("fixing Go files...")
+	return sggolangcilint.Fix(ctx)
```

Comment:
OK, you're saying we should remove the fix-specific config?

Reply:
Yes. 😄

Reply:
I can do that - I can override to serial mode with a flag instead - but technically can be done in a follow-up PR?

Reply:
It makes sense that `make go-lint` and `make go-lint-fix` uses the same config. Right?

Reply:
The point of the fix-specific config is to not run the non-fixing linters and speed up the run, and to apply serial mode instead of parallell mode - but probably better to have one single config like you say

Reply:
Ohh, the serial mode. Right!
Sure, we can do that as separate PR.

Reply:
I wonder if you can remove that particular part from the config alltogether and instead specify parallel vs serial on as CLI args...?